expo/expo
# [docs] Flatten meta tags in documentation page

# Why

It looks like Algolia only indexes the `latest` API docs. We haven't changed anything related to this on our end, but I noticed this in [the `next/head` docs](https://nextjs.org/docs/api-reference/next/head):

> `title`, `meta` or any other elements (e.g. `script`) need to be contained as direct children of the `Head` element, or wrapped into maximum one level of `<React.Fragment>` or arrays—otherwise the tags won't be correctly picked up on client-side navigations.

When switching between API pages I noticed the version isn't always updated, this change fixes that.


# How

Flattened the contents of `AlgoliaDocsearchMetaTags` by putting it in `DocumentationPage` directly.

# Test Plan

- `$ yarn dev`, open any versioned API page
- Inspect the `<head>` contents with Chrome
- Validate if you see a `<meta name="docsearch:version" ... />` tag
- Switch to different version
- `value` should now _always_ be the updated version
